app/views/requests/sponsor_request.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class='<%= dialog_column_class %>'>
+  <h1 id="dialogTitle">Share with your proxy group?</h1>
+
+  <%= render 'form_flashes' %>
+  <%= render 'searchworks_item_information' %>
+
+  <div class="<%= "#{label_column_offset_class} #{content_column_class}" %>">
+    <%= bootstrap_form_for(current_request) do |f| %>
+      <%= hash_to_hidden_fields(request_params) %>
+
+      <h2><%= render_user_information %></h2>
+      <%= f.form_group help: 'Anyone in your group can check the status of this request and pick it up when it’s available.' do %>
+        <%= f.button 'Yes, share with my group.', class: 'btn btn-default', value: 'true', name: 'request[proxy]' %>
```

Comment:
If you add `col-xs-12` to the button class these will behave like the similar buttons on the form.